### PR TITLE
Remove afunctional references to NetworkTables and ntproperty

### PR DESCRIPTION
**Pull request by ErikBoesen**

### CODE REVIEW

**Review by NathanielHorne — COMMENTED** [review]
Because we aren't doing network tables in this sample, and because the framework for rotation and arcade drive is here, I would say that this looks good.

**Review by NathanielHorne — APPROVED** [review]

File: robot/components/drive.py [hunk 1]
```diff
@@ -32,7 +28,7 @@ def move(self, y, rotation, square_inputs=False):
 
     def execute(self):
         """Actually drive."""
-        self.robot_drive.arcadeDrive(self.y, -self.rotation, square_inputs=False)
+        self.robot_drive.arcadeDrive(self.y, -self.rotation)
```
Comment (ArchdukeTim) [inline robot/components/drive.py:31]: something tells me the robot will rotate in the wrong direction...

Comment (ErikBoesen) [reply robot/components/drive.py:31]: It doesn't.

Comment (ArchdukeTim) [reply robot/components/drive.py:31]: Weird. ¯\\\_(ツ)_/¯

**Pull request merged**